common/preprocess.ts

```diff
@@ -62,6 +62,31 @@ const cleanActionStatus = (action, actionStatuses) => {
   return newStatus;
 };
 
+class DonutSector {
+  label: string;
+  color: string;
+  count: number;
+  includeInTotal: boolean;
+  constructor({
+    label,
+    color,
+    includeInTotal,
+  }: {
+    label: string;
+    color: string;
+    includeInTotal: boolean;
+  }) {
+    this.color = color;
+    this.label = label;
+    this.includeInTotal = includeInTotal;
+    this.count = 0;
+  }
+  incr() {
```

Comment:
Nitpick: I prefer clarity over saving characters in frontend code i.e. `increment()`